.github/workflows/build.yml

```diff
@@ -28,10 +28,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up JDK 1.8
+      - name: Set up JDK 1.11
         uses: actions/setup-java@v1
         with:
-          java-version: 8
+          java-version: 11
```

Comment:
Same here, should be the Java 8 version by default because it's used by the most of people.